src/components/cc-global-header.vue

```diff
@@ -128,11 +128,42 @@ export default defineComponent({
       ];
     } else {
       vm.menuLoading = true;
+      axios.interceptors.response.use(
+        undefined,
+        function retryApiRequest(error) {
```

Comment:
Let's move the function definition to the top of the JS Scope. E.g. right after the following block:

```js
if (process.env.NODE_ENV === "development") {
  var axios = require("axios");
} else {
  var axios = window.axios;
}

function retryApiRequest(error) {
  ...
}
```